packages/editor/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 - Add animated logo to preview interstitial screen.
 
+### Bug Fixes
+
+- Made preview interstitial text translatable.
+
```

Comment:
5.0.1 is the current version which was released on 2018-10-22. This dot point will need to move up to a new 5.0.2 section.

Reply:
Fixed after #10953